addons/hr/hr.py

```diff
@@ -317,8 +317,7 @@ def name_get(self, cr, uid, ids, context=None):
         return res
 
     def create(self, cr, uid, vals, context=None):
-        if context is None:
-            context = {}
+        context = dict(context) if context is not None else {}
         context['mail_create_nosubscribe'] = True
```

Comment:
I meant changing just the line 321 keeping the if above. rco will decide the style he prefers.